templates/style/base.scss

```diff
@@ -438,7 +438,7 @@ div.package-page-container {
             margin-top: 0;
         }
 
-        a.pure-menu-link {
+        .pure-menu-link {
```

Comment:
Hmm, why did this change?

Reply:
Because I wanted to use the style of this class in the "collapse" button.

Reply:
The classes don't entirely work on a non-link, if you click and drag it selects the text inside instead of dragging a button around like the other links.